alacritty_terminal/src/event_loop.rs

```diff
@@ -250,8 +250,10 @@ where
             }
         }
 
-        // Queue terminal redraw
-        self.event_proxy.send_event(Event::Wakeup);
+        if processed > 0 {
+            // Queue terminal redraw
+            self.event_proxy.send_event(Event::Wakeup);
+        }
```

Comment:
Is this necessary even after the event processing changes? I think we've said this only affects ConPTY, right?

Reply:
This was mostly WinPTY problem where it seems that `mio` always signals that a Windows named pipe should be read after child process exits, even if there is no actual data in the pipe, which causes UI loop to be flooded with those `Wakeup` events.

This seems to be the best way to fix it since `processed` holds the number of bytes that were actually read from the pipe inside [this read loop](https://github.com/jwilm/alacritty/blob/89ee0127c747a151f2763693af89d06314d8d74f/alacritty_terminal/src/event_loop.rs#L212).